src/content-script/in-dapp-notifications/styles.module.css

```diff
@@ -0,0 +1,160 @@
+.hstack,
+.vstack {
+  display: grid;
+}
+
+.hstack {
+  grid-auto-flow: column;
+  grid-auto-columns: minmax(min-content, max-content);
+  align-items: center;
+  justify-content: initial;
+}
+
+.vstack {
+  grid-template-columns: minmax(0, auto);
+  align-items: start;
+}
+
+.notification {
+  position: absolute;
+  top: 64px;
+  right: 18px;
+  z-index: 9999;
+  display: flex;
+  align-items: center;
+  padding: 12px;
+  gap: 12px;
+  border-radius: 12px;
+  background: #fff;
+  color: #16161a;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out;
```

Comment:
What is `transition` used for? The animation seems to be implemented with the `animation` property. Is this redundant?

Reply:
yes, these are leftovers, cleaned up ✅